tests/clickhouse/test_query_data.py

```diff
@@ -1,213 +1,12 @@
-from typing import Any, Sequence
-
-import pytest
 from snuba.clickhouse.columns import ColumnSet
 from snuba.clickhouse.formatter.query import format_query
 from snuba.clickhouse.query import Query as ClickhouseQuery
 from snuba.query import OrderBy, OrderByDirection, SelectedExpression
-from snuba.query.conditions import (
-    BooleanFunctions,
-    ConditionFunctions,
-    binary_condition,
-)
+from snuba.query.conditions import binary_condition
 from snuba.query.data_source.simple import Table
-from snuba.query.expressions import Column, CurriedFunctionCall, FunctionCall, Literal
+from snuba.query.expressions import Column, Literal
 from snuba.request.request_settings import HTTPRequestSettings
 
-test_cases = [
```

Comment:
I merged these tests into test_query_format. They were acting on the same queries, producing the same data structure (the formatted query) the only difference between the two was one assertion on the same result object.